test/services/AuditServiceSpec.scala

```diff
@@ -861,6 +861,73 @@ class AuditServiceSpec extends SpecBase with BeforeAndAfterEach {
     }
   }
 
+  "auditMaintainProfile" - {
+    val traderProfile        = TraderProfile(testEori, "XIUKIM47699357400020231115081800", None, None)
+    val updatedTraderProfile = TraderProfile(testEori, "XIUKIM47699357400020231115081801", None, None)
+
+    "return Done when built up an audit event and submitted it" in {
+
+      when(mockAuditConnector.sendEvent(any())(any(), any())).thenReturn(Future.successful(AuditResult.Success))
+
+      val fakeAuditEvent = DataEvent("source", "type")
+      when(mockAuditFactory.createMaintainProfileEvent(any(), any(), any())(any())).thenReturn(fakeAuditEvent)
+
+      val result =
+        await(auditService.auditMaintainProfile(traderProfile, updatedTraderProfile, AffinityGroup.Individual))
+
+      result mustBe Done
+
+      withClue("Should have supplied the trader profile and affinity group to the factory to create the event") {
+        verify(mockAuditFactory, times(1))
```

Comment:
just for future reference, yu do not need to add times(1) as this is the default.

Reply:
thanks Mauro, that's good spot, I have removed from currently added/updated tests but will raise separate ticket to remove from all the test cases.